tests/test4_rating_test.py

```diff
@@ -0,0 +1,129 @@
+import unittest
+import time
+from signUpDriver import signUp
+from createProjectDriver import createProject
+from evaluationDriver import evaluation
+from ratingDriver import rating
+
+class configure:
```

Comment:
I'm realizing that there are many places where you are passing around large tuples of configuration values. This tells me that you need to use object-oriented design.

I think we need a Configuration class, whose instance variables are these values like username, password,, etc. You would still have these all these configure functions but instead of returning tuples, they would return a Configuration object.

And, functions like createProject_attempt(username, password, projectname, projectdescription, studentFile, jsonFile) should take a Configuration object.

But we should talk about this together, because it looks like we might not want everything to go into the Configuration class. Only the most commonly used items

Reply:
Configuration class was created but our current implementation doesn't utilize it and it has since been removed from the test suite.